src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModInfo.java

```diff
@@ -150,9 +150,4 @@ public URL getUpdateURL() {
     {
         return this.logoFile;
     }
-
-    public boolean hasConfigUI()
```

Comment:
This is technically binary breaking, just mark it as deprecated and leave a comment there to remove it a future version

Reply:
I don't think mods are supposed to implement/override this, but OK I'll add it back